### DISCUSSION TITLE
Ao comecar com frontend é possivel participar de projetos?

### DISCUSSION THREAD

**Discussion by josekrauss020** [frontendbr/forum#2422] [Perguntas e Respostas]
Estou interessado em ganhar mais experiência na área de frontend, estou comecando na área e gostaria de saber algumas dicas. Tem como eu participar de projetos em conjunto?

**Comment by ninetails** [comment]
Uma maneira é por conta. [Tem o catálogo do Felipe Fialho sobre testes técnicos para frontend.](https://github.com/felipefialho/frontend-challenges) Pegue alguns e tente fazer por conta.

Outra maneira seria tentar ativamente participar de algo, voluntariamente ou dependendo tendo ajuda de custo. Se você faz curso ou graduação na área, pode ser que sua faculdade tenha alguma incubadora onde podem estar precisando de pessoas desenvolvedoras. Também pode ser que ache algo em ONGs e participando de comunidades da sua região.

---

**edit:** você também pode contribuir com _open source_. Tenta buscar projetos por aqui no GitHub com a etiqueta [`good first issue`](https://github.com/topics/good-first-issue). Mas aconselho também ir atrás de artigos sobre o tema antes de começar, pois nem sempre é só chegar lá e ajudar. Vários repositórios possuem certas regras, e contribuindo com código aberto você também já pega umas práticas de mercado.

**Comment by nazarepiedady** [comment]
@josekrauss020 em todos os projetos existem trabalhos que podem ser realizados profissionais juniores, consulte os projetos que mais são do teu interesse e procure peles questões (**issues**) do projeto o qua existe para ser feito ou pergunte para a comunidade em torno do projeto.
